src/core/public/application/application_service.tsx

```diff
@@ -243,6 +243,9 @@ export class ApplicationService {
       throw new Error('ApplicationService#setup() must be invoked before start.');
     }
 
+    const httpLoadingCount$ = new BehaviorSubject(0);
+    http.addLoadingCountSource(httpLoadingCount$);
```

Comment:
Wonder if this still makes sense now. But I'm very fine keeping it.

Reply:
I talked to @snide about this and he preferred having both.